ci-test.sh

```diff
@@ -25,8 +25,6 @@ else
     fi
 
     # Only testing it for one configuation to avoid running the same tests twice
-    echo "Checking compilation of wasm32 for all examples"
-    ./examples/check_all.sh
```

Comment:
It was actually helpful cause it was building wasm32 target. We should instead investigate why it fails to build now?

Reply:
./build_all.sh is building it anyway.

Reply:
But `build_all.sh` is not part of CI

Reply:
Fixed.